.github/workflows/build.yml

```diff
@@ -23,14 +23,10 @@ jobs:
         id: cache-node-modules
         uses: actions/cache@v3
         with:
-          path: '**/node_modules'
+          path: |
+           **/node_modules
+            ~/.cache/Cypress
           key: node-modules-${{ hashFiles('**/package-lock.json') }}
-      - name: Cache Cypress
-        id: cache-cypress
-        uses: actions/cache@v3
-        with:
-          path: ~/.cache/Cypress
-          key: cypress-cache
       - name: Cache eslint
```

Comment:
Could you add a `eslint-` restore key?
It is needed for the action to take the previous commit cache, use it and create a new one with the current comment `sha`.

```
      - name: Cache eslint
        id: cache-eslint
        uses: actions/cache@v3
        with:
          path: .eslintcache
          key: eslint-${{github.sha}}
          restore-keys: eslint-
```